sunpy/database/database.py

```diff
@@ -113,6 +113,51 @@ def __str__(self):  # pragma: no cover
         return errmsg.format(self.database_entry, self.tag_name)
 
 
+def split_database(source_database, destination_database, *query_string):
+    """
+    Queries the source database with the query string, and moves the
+    matched entries to the destination database. When this function is
+    called, the undo feature is disabled for both databases.
+
+    Parameters
+    ----------
+    source_database : sunpy.database.database.Database
+        A SunPy database object. This is the database on which the queries
+        will be made.
+    destination_database : sunpy.database.database.Database
+        A SunPy database object. This is the database to which the matched
+        entries will be moved.
+    query_string : list
+        A variable number of attributes that are chained together via the
+        boolean AND operator. The | operator may be used between attributes
+        to express the boolean OR operator.
+    
+    Example
+    -------
+    The function call in the following example moves those entries from
+    database1 to database2 which have Instrument = 'RHESSI' and
+    observation time falls in the range given by vso.attrs.Time.
+
+    >>> from sunpy.net import vso
+    >>> split_database(database1, database2, vso.attrs.Instrument('RHESSI'), vso.attrs.Time('2002-02-20 11:04:00', '2002-02-20 12:06:00'))
```

Comment:
@Cadair Is it still the case that the examples are executed by some travis job or somehow else? If yes, does this code have to be customized so that there are no undefined names (like ``database1`` and ``database2``)?